app/controllers/manage_results/subjects_controller.rb

```diff
@@ -3,6 +3,10 @@ def index
     @subjects = policy_scope(Subject).
       with_direct_assessments.
       sorted_by_number
+
+    if @subjects.count == 1
+      redirect_to action: "show", id: @subjects.first.id
```

Comment:
We should use the path helper here instead: `manage_results_subject_path(@subjects.first.id)`